src/map/handler/Map.Drag.js

```diff
@@ -96,12 +96,12 @@ export var Drag = Handler.extend({
 	},
 
 	_onDown: function () {
```

Comment:
Yes, absolutely. Just me being lazy here. Will fix if we think this is otherwise a good solution.